org.diffoscope.Diffoscope.yaml

```
@@ -0,0 +1,37 @@
+app-id: org.diffoscope.Diffoscope
+runtime: org.freedesktop.Sdk
+runtime-version: 18.08
```

This is a string, should be like this instead
```suggestion
runtime-version: '18.08'
```